Library/Homebrew/dev-cmd/bump-cask-pr.rb

```diff
@@ -75,17 +92,25 @@ def bump_cask_pr
     odie "This cask is not in a tap!" if cask.tap.blank?
     odie "This cask's tap is not a Git repository!" unless cask.tap.git?
 
-    new_version = unless (new_version = args.version).nil?
-      raise UsageError, "`--version` must not be empty." if new_version.blank?
+    new_version = Homebrew::NewVersion.new(
+      general: args.version,
+      arm:     args.version_arm,
+      intel:   args.version_intel,
+    )
 
-      new_version = :latest if ["latest", ":latest"].include?(new_version)
-      Cask::DSL::Version.new(new_version)
+    parse_new_version = lambda do |version|
```

Comment:
Could/should this be a regular method rather than a lambda?

Reply:
Moved to the NewVersion class.